Code/Framework/AzFramework/AzFramework/Asset/AssetCatalog.cpp

```diff
@@ -808,96 +808,118 @@ namespace AzFramework
     // note that this function is called from within a network job thread in order to update
     // the catalog as soon as new data is available, before other systems try to reach in and query
     // the results.  For this reason, it is a direct call and protects its structures with a lock.
-    void AssetCatalog::AssetChanged(AzFramework::AssetSystem::AssetNotificationMessage message)
+    void AssetCatalog::AssetChanged(const AZStd::vector<AzFramework::AssetSystem::AssetNotificationMessage>& messages, bool isCatalogInitialize)
```

Comment:
Is this a public API?

Reply:
This API is specifically marked internal